integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -263,6 +263,22 @@ public static boolean patchDomainResourceWithNewIntrospectVersion(
     return Domain.patchDomainResourceWithNewIntrospectVersion(domainUid, namespace);
   }
 
+  /**
+   * Get current introspectVersion for a given domain.
+   *
+   * @param domainUid domain id
+   * @param namespace namespace in which the domain resource exists
+   * @return String containing current introspectVersion
+   * @throws ApiException when getting domain resource fails
+   */
+  public static String getCurrentIntrospectVersion(String domainUid, String namespace) throws ApiException {
+    LoggingFacade logger = getLogger();
+    oracle.weblogic.domain.Domain domain = Domain.getDomainCustomResource(domainUid, namespace);
```

Comment:
no implementation in TestActions.java. create a method in Domain class and call that from here.

Reply:
changed